codalab/worker/rest_client.py

```diff
@@ -27,7 +28,7 @@ class RestClient(object):
     """
     A dictionary of additional headers to send along with HTTP requests.
     """
-    _extra_headers = {}
+    _extra_headers: Dict[Any, Any] = {}
```

Comment:
Can the types be more specific?  At least the key is a string, no?

Reply:
sure, I can do it.